chain4energy/chain.json

```diff
@@ -678,6 +695,11 @@
       "url": "https://explorer.apeironnodes.com/chain4energy",
       "tx_page": "https://explorer.apeironnodes.com/chain4energy/transactions/${txHash}"
     },
+    {
+      "kind": "explorer",
```

Comment:
I think kind here should be the host (DTEAM)

Reply:
yep

just changed it, thanks